themes/default/content/blog/converting-bicep-to-pulumi/index.md

```diff
@@ -10,6 +10,8 @@ tags: ["community", "bicep", "pulumi", "convert"]
 
 Bicep is a DSL developed by Microsoft to simplify the authoring of ARM templates and deploy resources to Azure. Today I will be sharing with you a new Pulumi converter plugin that I have been working on that converts Bicep code to any of the supported Pulumi languages.
 
+{{< youtube "VNj1EyQ7wug?rel=0" >}}
+
 <!--more-->
```

Comment:
I'd probably suggest putting the video after the jump, as `<!--more-->` demarcates what shows up on the index page vs. the post page:

```suggestion
<!--more-->

{{< youtube "VNj1EyQ7wug?rel=0" >}}
```